src/time.rs

```diff
@@ -0,0 +1,302 @@
+/*! Time structures.
+
+The `time` module contains structures used to represent both
+absolute and relative time.
+
+ - [Instant] is used to represent absolute time.
+ - [Duration] is used to represet relative time.
+
+[Instant]: struct.Instant.html
+[Duration]: struct.Duration.html
+*/
+
+use core::{ops, fmt};
+
+#[cfg(feature = "std")]
+const NANOS_PER_MILLI: u32 = 1_000_000;
```

Comment:
One last nit: let's just embed this one too. Meanwhile let me fix the build failure.

Reply:
I've released managed 0.5.1 which should fix the build failure.

Reply:
Is that what is happening in #116 as well?

Reply:
I think so